bedrock/foundation/templates/foundation/trademarks/list.html

```diff
@@ -15,127 +15,113 @@ <h1 class="page-title">{{ _('List of Mozilla Trademarks') }}</h1>
+      <th>{{ _('SUMO logo') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Rust logo') }}™" src="{{ static('img/trademarks/rust-logo.png') }}" /></td>
-      <th>{{ _('Rust logo') }}&trade;</th>
+      <td><img width="100" height="100" alt="{{ _('QMO logo') }}™" src="{{ static('img/trademarks/qmo-logo-tm.png') }}" /><span>&trade;</span></td>
+      <th>{{ _('QMO logo') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Cargo logo') }}™" src="{{ static('img/trademarks/cargo-logo.png') }}" /></td>
-      <th>{{ _('Cargo logo') }}&trade;</th>
+      <td><img width="100" height="100" alt="{{ _('G-Fox logo') }}®" src="{{ static('img/trademarks/gfox-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{{ _('G-Fox logo') }}</th>
     </tr>
-  </table>
-
-  <table id="marks">
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Firefox') }}®" src="{{ static('img/trademarks/firefox_wordmark-only_RGB.png') }}" /></td>
-      <th>{{ _('Firefox') }}&reg;</th>
+      <td><img width="100" height="100" alt="{{ _('Bugzilla logo') }}™" src="{{ static('img/trademarks/bugzilla-tm.png') }}" /><span>&trade;</span></td>
+      <th>{{ _('Bugzilla logo') }}</th>
     </tr>
-      <tr>
-        <td><img width="100" height="100" alt="{{ _('Firefox® OS') }}" src="{{ static('img/trademarks/firefox-os_wordmark-only_RGB.png') }}" /></td>
-        <th>{{ _('Firefox&reg; OS') }}</th>
-      </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Thunderbird') }}®" src="{{ static('img/trademarks/thunderbird-r.png') }}" /></td>
-      <th>{{ _('Thunderbird') }}&reg;</th>
+      <td><img width="100" height="100" alt="{{ _('Sunbird logo') }}®" src="{{ static('img/trademarks/sunbird-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{{ _('Sunbird logo') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Mozilla') }}®" src="{{ static('img/trademarks/mozilla-r.png') }}" /></td>
-
-      <th>{{ _('Mozilla') }}&reg;</th>
-    </tr>
-    <tr>
-      <td><img width="100" height="100" alt="{{ _('Camino') }}®" src="{{ static('img/trademarks/camino-r.png') }}" /></td>
-      <th>{{ _('Camino') }}&reg;</th>
+      <td><img width="100" height="100" alt="{{ _('SeaMonkey logo') }}®" src="{{ static('img/trademarks/seamonkey-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{{ _('SeaMonkey logo') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Mozilla Communities') }}™" src="{{ static('img/trademarks/community-wordmark.png') }}" /></td>
-
-      <th>{{ _('Mozilla Communities') }}&trade;</th>
+      <td><img width="100" height="100" alt="{{ _('Foxkeh logo') }}®" src="{{ static('img/trademarks/foxkeh-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{% trans %}Foxkeh logo<br /><small>This trademark is licensed under the Creative Commons Attribution-Non-Commercial-NoDerivs 2.1 Japan.</small>{% endtrans %}</th>
```

Comment:
That would be better. I did not change it and it is as it was before I believe but, I like your suggestion and, this is not currently translated so, will not be breaking l10n